test/13_arraycommon.jl

```diff
@@ -1642,4 +1642,18 @@ end
     @test levelcode.(x) isa Vector{Union{Missing,Int16}}
 end
 
+@testset "fill()" begin
+    for ordered in (false, true), dims in ([1], [2, 3], [(2, 3)], [])
+        x = CategoricalArray{String, 1, UInt8}(["a", "b", "c"],
```

Comment:
do you think it is worth to add a second test of the same, but where `x` contains a `missing` instead of `"c"`?

Reply:
I've thought about it, but I don't think so since the `CategoricalValue` doesn't reflect that at all.

Reply:
That is the point, as it is not obvious. But for this test it is OK as is